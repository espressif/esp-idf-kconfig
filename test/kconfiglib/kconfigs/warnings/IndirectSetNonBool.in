mainmenu "Indirectly set values in non-bool configs"

    config NONBOOL
        int "Non-bool config"
        default 0
        set A=1
        set default A=42
        select B
        imply C

    config A
        int "A"
        default 0

    config B
        bool "B"
        default n

    config C
        bool "C"
        default n
