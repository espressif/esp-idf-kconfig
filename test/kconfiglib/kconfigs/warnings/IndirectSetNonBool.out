# default:
CONFIG_NONBOOL=0
# default:
CONFIG_A=0
# default:
# CONFIG_B is not set
# default:
# CONFIG_C is not set
